vello/src/scene.rs

```diff
@@ -400,7 +400,7 @@ impl<'a> DrawGlyphs<'a> {
     }
 
     /// Sets the normalized design space coordinates for a variable font instance.
-    pub fn normalized_coords(mut self, coords: &[NormalizedCoord]) -> Self {
+    pub fn normalized_coords(mut self, coords: impl Iterator<Item = i16>) -> Self {
```

Comment:
We could require `ExactSizeIterator` here if we want to.

Reply:
I don’t think it would hurt to have a `pub type NormalizedCoord = i16` in vello_encoding

Reply:
What does `ExactSizeIterator` win us?

Reply:
I think we definitely should document this slightly better, even if just to say "this will be provided by your text layout implementation". The current format is quite unactionable.

Reply:
> What does `ExactSizeIterator` win us?

Potentially slightly cheaper allocation. But I suspect `size_hint` will work even without `ExactSizeIterator`, and that even if it doesn't the cost will be small.